examples/ae_examples/cvae_dim_example/client.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+from pathlib import Path
+from typing import Sequence, Tuple
+
+import flwr as fl
+import torch
+import torch.nn as nn
+import torchvision.transforms as transforms
+from flwr.common.typing import Config
+from torch.nn.modules.loss import _Loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader
+
+from examples.models.mnist_model import MnistNet
+from fl4health.clients.basic_client import BasicClient
+from fl4health.preprocessing.autoencoders.dim_reduction import CvaeFixedConditionProcessor
+from fl4health.utils.load_data import load_mnist_data
+from fl4health.utils.metrics import Accuracy, Metric
+from fl4health.utils.random import set_all_random_seeds
+from fl4health.utils.sampler import DirichletLabelBasedSampler
+
+
+class CvaeDimClient(BasicClient):
+    def __init__(self, data_path: Path, metrics: Sequence[Metric], DEVICE: torch.device, condition: torch.Tensor):
+        BasicClient.__init__(self, data_path, metrics, DEVICE)
```

Comment:
Perhaps there is a reason to explicitly call the parent init, but I think you can do this more generically with `super().__init__(data_path, metrics, device)`

Reply:
No, there is no reason for that. This is the remaining trace from the previous design :joy:. I will change it.